java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1649,11 +1648,17 @@ public void xadd(BaseClient client) {
                             .getSingleValue());
         }
 
-        /**
-         * TODO add test to XTRIM on maxlen expect( await client.xtrim(key, { method: "maxlen",
-         * threshold: 1, exact: true, }), ).toEqual(1); expect(await client.customCommand(["XLEN",
-         * key])).toEqual(1);
-         */
+        // test xtrim to remove 1 element
+        assertEquals(1L, client.xtrim(key, new MaxLen(true, 1)).get());
+
```

Comment:
can you run xlen here?

Reply:
xlen is not yet implemented.  Want me to add a xlen todo?

Reply:
yes or custom command + todo